viewing_party/party.py

```diff
@@ -1,23 +1,248 @@
+
+# -----------------------------------------
 # ------------- WAVE 1 --------------------
+# -----------------------------------------
 
 def create_movie(title, genre, rating):
-    pass
+    #create a dictionary with the keys 'title', 'genre', 'rating', and the values the values passed in
+    if title and genre and rating:

```

Comment:
Nice check for truthy-ness! Great intuition that if they're not all truthy, omitting a `return` will effectively return `None`.